src/validation/permissionsValidation.js

```diff
@@ -0,0 +1,69 @@
+import { validationResult } from 'express-validator';
+import models from '../models';
+import errorAssignment from '../utils/errorAssignment';
+
+const { roles } = models;
+
+const validateRole = async (request) => {
+  let errors = {};
+  const { affectedRole } = request.body;
+  if (!affectedRole) errors = { ...errorAssignment('Role input is compulsory', 'affectedRole') };
+  if (!/^\d+$/.test(affectedRole)) errors = { ...errorAssignment('Role input must be a number', 'affectedRole') };
+
+  if (!errors.affectedRole && affectedRole) {
+    const userRole = await roles.findOne({
+      where: { id: affectedRole },
+      raw: true
+    });
+    if (!userRole) {
+      errors = { ...errorAssignment('Check role input and try again', 'affectedRole') };
```

Comment:
This error message isn't descriptive enough. Maybe we can say `Role is invalid` or `Role does not exist` or something else that makes the user know why the role he/she inputted did not work

Reply:
I have made the changes as instructed